### PR TITLE
7.2 Developer Docs | Data Engine | LRDOCS-6696

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6696. @jeyvison please do an initial review of 01-intro.markdown and 02-creating-data-definitions.markdown. Let me know how useful this type of documentation would be for third party developers. cc @brunofarache.

### CODE REVIEW

File: developer/tutorials/articles/data-engine/01-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+Collection.
+
+## Data Views
+
+There's another important piece to the Data Engine framework: displays.
+
+Once you have a Data Definition and a Data Collection, it's common to need
+control over display. Do this with FreeMarker templates, like in the Dynamic
+Data Lists application.
+
```
Comment (rbohl) [inline developer/tutorials/articles/data-engine/01-intro.markdown]: Stop reviewing here. My earlier notes follow, that I need to delete from the markdown.

**Comment by marcelabc** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: developer/tutorials/articles/data-engine/01-intro.markdown [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    Validation: Both are required fields.
+
+    Permissions: Admins should be able to manage these with full CRUD operation support
+
+4.  Create the Guestbook Data Record Collection
+
+    Each Data Record collection represents a different Guestbook
+
+    Admins can do the full set of CRUD operations on these, from an
+    administrative UI (is that possible with data engine?)
```
Comment (marcelabc) [inline developer/tutorials/articles/data-engine/01-intro.markdown]: Is that possible with data engine? Answer: Not today.

File: developer/tutorials/articles/data-engine/01-intro.markdown [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+5.  Creating Data Layouts
+
+    Expose the administrative functions for the Guestbook Data Record
```
Comment (marcelabc) [inline developer/tutorials/articles/data-engine/01-intro.markdown]: 'Expose the administrative functions for the Guestbook Data Record Collections'

This is nothing related to the layout. The layout is just how the data definition will be displayed to the user (i.e., whether the fields will be in the same column or not). The user is going to fill, and that information is going to be a Data Record to that particular Data Definition and Data Record Collection.

File: developer/tutorials/articles/data-engine/02-creating-data-definitions.markdown [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+```sh
+curl -i -X POST \
+  http://localhost:8080/o/data-engine/v1.0/data-record-collections/39212/data-records \
+  -H 'Authorization: Basic dGVzdEBsaWZlcmF5LmNvbTp0ZXN0' \
+  -H 'Content-Type: application/json' \
+  -d '{
+
+        "dataRecordValues": {
+
+            "name": "product",
```
Comment (marcelabc) [inline developer/tutorials/articles/data-engine/02-creating-data-definitions.markdown]: I believe it is "key", but I going to double check

**Comment by marcelabc** [comment]
Hi @rbohl , 

I just finished reviewing your PR. If you need any help ou an update, please talk to me :)

**Comment by rbohl** [comment]
Thanks @marcelabc . I am pushing some updates based on your feedback, and will send a new PR.

**Pull request closed without merging**